### PR TITLE
Short read connector tool

### PR DESCRIPTION
**Pull request by cmonjeau**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/short-read-connector/build.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+# decrease RAM needed
+sed -i.bak 's/make -j/make -j1/' INSTALL
+sed -i.bak 's/2> log_linker_err//' test/simple_test.sh
+
+# change run_discoSnp++.sh deps path
+sed -i.bak 's/\$EDIR\/bin/\$EDIR/' short_read_connector.sh
```
Comment (bgruening) [inline recipes/short-read-connector/build.sh]: @cmonjeau you can also write this in the following way: `'s|\$EDIR/bin|\$EDIR|'`

File: recipes/short-read-connector/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib {{CONDA_ZLIB}}*
+    - git
```
Comment (bgruening) [inline recipes/short-read-connector/meta.yaml]: why is git needed?

Comment (cmonjeau) [reply recipes/short-read-connector/meta.yaml]: Hi @bgruening. Thanks for the review.

Inside the INSTALL file, there are git commands to get external repositories but it seems the dependency is not needed :)

File: recipes/short-read-connector/build.sh (hunk 1 above)
Comment (cmonjeau) [reply recipes/short-read-connector/build.sh]: Done

**Pull request merged**